engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -284,6 +279,20 @@ public void addRMDelegationToken(final String renewer, final Credentials credent
       credentials.addToken(token.getService(), token);
     }
 
+    public InetSocketAddress getRMHAAddress(String rmId)
+    {
+      YarnConfiguration yarnConf;
+      if (conf instanceof YarnConfiguration) {
+        yarnConf = (YarnConfiguration)conf;
+      } else {
+        yarnConf = new YarnConfiguration(conf);
+      }
+      yarnConf.set(ConfigUtils.RM_HA_ID, rmId);
+      InetSocketAddress socketAddr = yarnConf.getSocketAddr(YarnConfiguration.RM_ADDRESS, YarnConfiguration.DEFAULT_NM_ADDRESS, YarnConfiguration.DEFAULT_RM_PORT);
```

Comment:
It looks like a typo here. Should it be YarnConfiguration.DEFAULT_RM_ADDRESS, and not YarnConfiguration.DEFAULT_NM_ADDRESS.